pages/events/[id].tsx

```diff
@@ -0,0 +1,232 @@
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import { useGetEventByIdQuery } from 'generated/graphql';
+import { Typography } from 'components/atoms/Typography/Typography';
+import { useState } from 'react';
+import Camera from 'components/icons/Camera.svg';
+import Star from 'components/icons/Star.svg';
+import UserIcon from 'components/icons/UserIcon.svg';
+import Ticket from 'components/icons/Ticket.svg';
+
+const EventDetailsPage = () => {
+	const router = useRouter();
+	const id = router.query.id as string;
+	const { data, loading } = useGetEventByIdQuery({ variables: { id } });
+	const [isEvent, setIsEvent] = useState(true);
```

Comment:
What you think if we keep here a string of current view, not a boolean. It wuold be much easier to add a new view that way. You can use [template literal](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html) to be sure to set a good string value.

Reply:
String of current view is now included